packages/calypso-build/webpack.config.js

```diff
@@ -21,6 +21,7 @@ const { cssNameFromFilename, shouldTranspileDependency } = require( './webpack/u
  */
 const isDevelopment = process.env.NODE_ENV !== 'production';
 const cachePath = path.resolve( '.cache' );
+const shouldCheckForDuplicatePackages = ! process.env.DISABLE_DUPLICATE_PACKAGE_CHECK;
```

Comment:
Keeps the duplicated package checker enabled by default, but allows us to disable it to avoid webpack 5 deprecation warnings. Inspired by https://github.com/Automattic/wp-calypso/pull/45858